test/src/main/java/org/openjdk/skara/test/TestPullRequestStore.java

```diff
@@ -124,9 +129,16 @@ public void setSourceRef(String sourceRef) {
 
     public void setDraft(boolean draft) {
         this.draft = draft;
+        if(!draft){
```

Comment:
```suggestion
        if (!draft) {
```

Reply:
> Nice work! I have some comments, but nothing really major.
> 
> I put in some style corrections in some places. Please try to match the existing style with whitespace between keyword and opening paren as well as closing paren and opening brace.

My bad, I have the habit of using intellij to automatically adjust the format, but this time I forgot at some places, I will fix it.